dotnet/src/AutoGen.Core/Function/FunctionAttribute.cs

```diff
@@ -98,7 +98,7 @@ public static implicit operator AIFunctionMetadata(FunctionContract contract)
                 [NamespaceKey] = contract.Namespace,
                 [ClassNameKey] = contract.ClassName,
             },
-            Parameters = [.. contract.Parameters?.Select(p => (AIFunctionParameterMetadata)p)],
+            Parameters = [.. contract.Parameters?.Select(p => (AIFunctionParameterMetadata)p)!],
```

Comment:
Is this change needed?

Reply:
I don't know where that one came from. I'll back it out.

Reply:
hmm, mine won't build without it now. "Dereference of possibly null...."